frontend/scss/organisms/_o-custom-tours-viewer.scss

```diff
@@ -40,6 +39,117 @@
   }
 }
 
+.aic-ct-grid-card,
+.aic-ct-viewer {
+  .f-headline-editorial {
+    font-weight: 300;
+
+    &::before {
+      padding-top: 0;
+    }
+
+    &::after {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.m-cta-banner--aic-ct,
+.aic-ct-viewer {
+  .f-headline {
+    &::before {
+      padding-top: 0;
+    }
+
+    &::after {
+      margin-bottom: 0;
+    }
+  }
+}
+
+.aic-ct-viewer {
+  .f-subheading-1,
+  .f-secondary,
+  .f-body,
+  .f-tertiary,
+  .f-link,
+  .f-body-editorial-emphasis,
+  .f-quote {
+    &::before {
+      padding-top: 0;
+    }
+
+    &::after {
+      margin-bottom: 0;
+    }
+  }
+
+  .f-quote {
+    font-weight: 300;
+    letter-spacing: 0.015em
```

Comment:
I think there should be a semicolon on the end of this line. It might be worth checking the rest of your changes for this.